components/Dialogs/tests/snapshot/MDCAlertControllerActionsTests.m

```diff
@@ -199,4 +201,203 @@ - (void)testAutomaticChangeToVerticalLayoutForLongMediumEmphasisActionsInRTL {
+  [self addSecondShortActionWithEmphasis:MDCActionEmphasisLow];
+  [self.alertController applyThemeWithScheme:self.containerScheme2019];
+
+  // When
+  MDCAlertControllerView *view = (MDCAlertControllerView *)self.alertController.view;
+  view.actionsHorizontalAlignmentInVerticalLayout = MDCContentHorizontalAlignmentLeading;
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.alertController.view];
+}
+
+// Vertical Layout | Low Emphasis | Justified Alignment
+- (void)testLowEmphasisActionsAreJustifiedInVerticalLayout {
+  // Given
+  [self addFirstLongActionWithEmphasis:MDCActionEmphasisLow];
+  [self addSecondShortActionWithEmphasis:MDCActionEmphasisLow];
+  [self.alertController applyThemeWithScheme:self.containerScheme2019];
+
+  // When
+  MDCAlertControllerView *view = (MDCAlertControllerView *)self.alertController.view;
+  view.actionsHorizontalAlignmentInVerticalLayout = MDCContentHorizontalAlignmentJustified;
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.alertController.view];
+}
+
+// Vertical Layout | Medium Emphasis | Trailing Alignment
+- (void)testMediumEmphasisActionsAreTrailingInVerticalLayout {
+  // Given
+  [self addFirstLongActionWithEmphasis:MDCActionEmphasisMedium];
+  [self addSecondShortActionWithEmphasis:MDCActionEmphasisMedium];
+  [self.alertController applyThemeWithScheme:self.containerScheme2019];
+
+  // When
+  MDCAlertControllerView *view = (MDCAlertControllerView *)self.alertController.view;
+  view.actionsHorizontalAlignmentInVerticalLayout = MDCContentHorizontalAlignmentTrailing;
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.alertController.view];
+}
+
+// Vertical Layout | Medium Emphasis | Leading Alignment
+- (void)testMediumEmphasisActionsAreLeadingInVerticalLayout {
+  // Given
+  [self addFirstLongActionWithEmphasis:MDCActionEmphasisMedium];
+  [self addSecondShortActionWithEmphasis:MDCActionEmphasisMedium];
+  [self.alertController applyThemeWithScheme:self.containerScheme2019];
+
+  // When
+  MDCAlertControllerView *view = (MDCAlertControllerView *)self.alertController.view;
+  view.actionsHorizontalAlignmentInVerticalLayout = MDCContentHorizontalAlignmentLeading;
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.alertController.view];
+}
+
+// Vertical Layout | Medium Emphasis | Justified Alignment
+- (void)testMediumEmphasisActionsAreJustifiedInVerticalLayout {
+  // Given
+  [self addFirstLongActionWithEmphasis:MDCActionEmphasisMedium];
+  [self addSecondShortActionWithEmphasis:MDCActionEmphasisMedium];
+  [self.alertController applyThemeWithScheme:self.containerScheme2019];
+
+  // When
+  MDCAlertControllerView *view = (MDCAlertControllerView *)self.alertController.view;
+  view.actionsHorizontalAlignmentInVerticalLayout = MDCContentHorizontalAlignmentJustified;
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.alertController.view];
+}
+
+#pragma mark - Helpers
+
+- (void)addFirstLongActionWithEmphasis:(MDCActionEmphasis)emphasis {
```

Comment:
Instead of having three methods would it be better to have one that takes an emphasis and title?

Reply:
Removed one of them.
They make the tests more readable this way.
I also prefer to  keep the title in one place.